dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -189,26 +192,28 @@ private static void printStaticLinkageErrors(
 
   /**
    * Finds linkage errors in the input classpath and generates a static linkage check report.
+   *
+   * @return a static linkage check report for the input class path
```

Comment:
This just repeats what's in the summary, so you can leave it out.

Reply:
Removed.